library/core/src/pin.rs

```diff
@@ -387,6 +725,13 @@ use crate::hash::{Hash, Hasher};
 use crate::marker::{Sized, Unpin};
 use crate::ops::{CoerceUnsized, Deref, DerefMut, DispatchFromDyn, Receiver};
 
+use crate::{
```

Comment:
I assume these are for intra-doc links. You'll need to `allow(unused_imports)`, rustc doesn't know that rustdoc uses these.
```suggestion
#[allow(unused_imports)]
use crate::{
```

Reply:
Ah! I had used `#[cfg(doc)]` but that made one of the CI passes really confused.

Reply:
Yeah, cfg(doc) breaks the links when they're exported, it's not set for dependencies.